packages/design-system/package.json

```diff
@@ -0,0 +1,153 @@
+{
+  "name": "@livechat/design-system",
+  "version": "0.5.6",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "LiveChat Design System React components with css styles",
+  "homepage": "https://livechat.github.io/design-system/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/livechat/design-system"
+  },
+  "typings": "src/interfaces/index.d.ts",
+  "main": "dist/design-system.cjs.js",
+  "module": "dist/design-system.es.js",
+  "jsnext:main": "dist/design-system.es.js",
+  "style": "dist/design-system.css",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "scripts": {
+    "start": "rollup -c --watch",
+    "build": "rollup -c",
+    "prebuild": "rimraf dist",
+    "build:cdn": "npm run build && mkdir -p dist/cdn && cp dist/design-system.css dist/cdn/styles.css && cp dist/cdn/styles.css dist/cdn/styles-v$npm_package_version.css",
+    "prepare": "npm run build",
+    "test": "jest",
+    "test:update": "jest -u",
+    "test:watch": "jest --watchAll",
+    "test:coverage": "jest --coverage",
+    "eslint": "eslint --ext .js,.ts src/ ",
+    "eslint:fix": "eslint --ext .js,.ts src/ --fix "
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/prop-types": "^15.5.6",
+    "@types/react": "^16.4.15",
+    "@types/react-dom": "^16.0.8",
+    "autoprefixer": "^9.0.1",
+    "babel-cli": "^6.26.0",
+    "babel-eslint": "^8.2.6",
+    "babel-jest": "^23.4.2",
+    "babel-loader": "^7.1.1",
+    "babel-plugin-css-modules-transform": "^1.6.1",
+    "babel-plugin-external-helpers": "^6.22.0",
+    "babel-plugin-react-svg": "^2.1.0",
+    "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-plugin-transform-imports": "^1.5.0",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
+    "babel-plugin-transform-react-jsx": "^6.24.1",
+    "babel-plugin-transform-react-remove-prop-types": "^0.4.15",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-es2015-rollup": "^3.0.0",
+    "babel-preset-react": "^6.3.13",
+    "caniuse-lite": "1.0.30000974",
+    "cross-env": "^5.2.0",
+    "css-loader": "^2.1.1",
+    "cssnano": "^4.0.5",
+    "enzyme": "^3.3.0",
+    "enzyme-adapter-react-16": "^1.1.1",
+    "enzyme-to-json": "^3.3.4",
+    "eslint": "^4.19.1",
```

Comment:
same story for the eslint (move it to the root)

Reply:
done at 13352591223f2ba618e2916566e485b92d3e402f